### PR TITLE
2fa backup codes test coverage

### PR DESCRIPTION
**Pull request by ChristophWurst**
Apparently the activity provider was completely untested, so I've added some tests for it. While at it, I also tried to increase the overall coverage of this app. Interestingly, the coverage is sometimes measured wrongly, e.g. https://codecov.io/gh/nextcloud/server/src/203436b16be40109131720f06ca39e393c373bae/apps/twofactor_backupcodes/lib/Db/BackupCodeMapper.php is actually covered by an integration test https://github.com/nextcloud/server/blob/6c7a7e0c1a9c8c38ed246176ee5c491620d9630c/apps/twofactor_backupcodes/tests/Integration/Db/BackupCodeMapperTest.php.

### CODE REVIEW

File: apps/twofactor_backupcodes/appinfo/app.php [hunk 1]
```diff
@@ -19,4 +19,7 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *
  */
+
+// @codeCoverageIgnoreStart
 OC_App::registerPersonal('twofactor_backupcodes', 'settings/personal');
+// @codeCoverageIgnoreEnd
```
Comment (blizzz) [inline apps/twofactor_backupcodes/appinfo/app.php]: make sure that your files have an empty line at the end

File: apps/twofactor_backupcodes/settings/personal.php [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <?php
-
+// @codeCoverageIgnoreStart
 $tmpl = new \OCP\Template('twofactor_backupcodes', 'personal');
 
 return $tmpl->fetchPage();
+// @codeCoverageIgnoreEnd
```
Comment (blizzz) [inline apps/twofactor_backupcodes/settings/personal.php]: see above

**Review by blizzz — COMMENTED** [review]
Looks good, just a formality

**Review by blizzz — APPROVED** [review]

File: apps/twofactor_backupcodes/settings/personal.php (hunk 2 above)
Comment (blizzz) [inline apps/twofactor_backupcodes/settings/personal.php]: ah, this one still has it

**Review by LukasReschke — APPROVED** [review]

**Pull request merged**